nifi-nar-bundles/nifi-cassandra-bundle/nifi-cassandra-services-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,327 @@
+nifi-cassandra-services-nar
+Copyright 2016 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+******************
+Apache Software License v2
+******************
+
+The following binary components are provided under the Apache Software License v2
+
+  (ASLv2) DataStax Java Driver for Apache Cassandra - Core
+      The following NOTICE information applies:
+        DataStax Java Driver for Apache Cassandra - Core
+        Copyright (C) 2012-2017 DataStax Inc.
+
+  (ASLv2) Apache Avro
+      The following NOTICE information applies:
+        Apache Avro
+        Copyright 2009-2017 The Apache Software Foundation
+
+  (ASLv2) Jackson JSON processor

```

Comment:
Same case here. I don't think this dependency is used anywhere in the CS' POM.

Reply:
It is referenced in `mvn dependency:tree` on your services NAR.

Reply:
Understood.